src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">
+            Surname:
+            <input
+              class="form-field"
+              type="text"
+              name="surname"
+              id="surname"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
```

Comment:
div wrapper is redundant. use class="form-field" to a label

Reply:
Does it mean that we don't need a div wrapper? I thought it helps to start each input on the new line. I moved class="form-field" to label tags, and changed .form-field:not(:last-child) in CSS, however, 10 px margins don't apply at all. Should all labels have this class?